ml-models/training.py

```diff
@@ -13,7 +13,7 @@
 from utils.plotting import plot_loss_history
 
 
-def train(model, X_train, y_train, X_val, y_val, batch_size, learning_rate, epochs, save_path=None, grad_clipping=None):
+def train(model, X_train, y_train, X_val, y_val, batch_size, learning_rate, epochs, y_scaler, save_path=None, grad_clipping=None, ):
```

Comment:
```suggestion
def train(model, X_train, y_train, X_val, y_val, batch_size, learning_rate, epochs, y_scaler, save_path=None, grad_clipping=None):
```
sjovt at den ville acceptere det